packages/manager/modules/sms/src/telecom-sms.scss

```diff
@@ -1,6 +1,6 @@
 .telecom-sms {
```

Comment:
This file should be renamed as `index.scss` in order to keep consistency across modules. Isn't it?